dev/DynamicDependency/DynamicDependencyLifetimeManagerShadow/winmain.cpp

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+const HRESULT WINDOWSAPPSDK_DDLM_SHUTDOWN{ MAKE_HRESULT(SEVERITY_SUCCESS, FACILITY_ITF, 0x8000) };
+const HRESULT WINDOWSAPPSDK_DDLM_CALLER_TERMINATED{ MAKE_HRESULT(SEVERITY_SUCCESS, FACILITY_ITF, 0x8001) };
+const HRESULT WINDOWSAPPSDK_DDLM_CALLER_NOT_FOUND{ MAKE_HRESULT(SEVERITY_SUCCESS, FACILITY_ITF, 0x8002) };
+
+int WINAPI WinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, PSTR /*lpCmdLine*/, int /*nCmdShow*/)
```

Comment:
>WinMain [](http://example.com/codeflow?start=11&length=7)

wondering how this app informs deployment WAS is in use?

Reply:
It's a DesktopBridge application so it's a packaged process, which suffices to inform Deployment a running process has a dependency on WARfwk and thus WARfwk is in use.

The PackagedCOM OOP Server serves the same purpose. A process running with package identity and a manifested dependency on WARfwk.

See the added application Id="DDLM" in `test/DynamicDependency/data/DynamicDependencyLifetimeManager.Msix/appxmanifest.xml`. That's just the test package; adding an equivalent snippet to the DDLM package's manifest in the Aggregator repo